src/bucket.js

```diff
@@ -131,7 +145,11 @@ export default class Bucket {
    */
   createCollection(id, options={}) {
     const reqOptions = this._bucketOptions(options);
-    const request = requests.createCollection(id, reqOptions);
+    const { permissions, data={} } = reqOptions;
+    data.id = id;
+    const path = id ? endpoint("collection", this.name, id) :
+                      endpoint("collections", this.name);
```

Comment:
In theory simply `const path = endpoint("collection", this.name, id);` should get us covered.